.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,45 @@
 ---
-name: Bug report
+name: Bug Report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-# Please follow the general troubleshooting steps first:
+Note: If the bug affects multiple games, please open an issue for each game with the game in the title.
 
-- [ ] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and completed successfully the setup wizard.
+# Bug Reports
```

Comment:
No need for a primary header if it shows up in the issue contents

Reply:
```suggestion
```